docs/sources/configure/escalation-chains-and-routes/index.md

```diff
@@ -88,8 +88,11 @@ via the method configured in their user profile.
 * `Notify Slack User Group` - send a notification to each member of a slack user group. These users will be notified
 via the method configured in their user profile.
 * `Trigger outgoing webhook` - trigger an [outgoing webhook].
-* `Notify users one by one (round robin)` - each notification will be sent to a group of
-users one by one, in sequential order in [round robin fashion](https://en.wikipedia.org/wiki/Round-robin_item_allocation).
+* `Notify users one by one (round robin)` - notify users sequentially, cycling through users for **different alert groups**.
+Example: if users A, B, and C are in the list, the first alert group notifies A, the second alert group notifies B, and
+the third alert group notifies C.
+To notify multiple users within the same alert group until someone acknowledges, use `Notify users` policies with
+`Wait` policies between them in the escalation chain.
```

Comment:
FWIW, I was answering a similar thing just yesterday:

> the RR step means that every time that step is triggered, the next user will be paged. So, during an escalation it will page only one user, but next time an alert group fires, it will page the next one. What you could do, for example, is add a "Repeat escalation" step, which will loop through the escalation again, and re-trigger the step (paging the following user in the round)

Maybe also worth noting that users are sorted alphabetically by their username?

Reply:
good call 👍 added in https://github.com/grafana/oncall/pull/4662/commits/649164254a9358b32c2ade52d14451681d10c491